libs/artifacts/types.d.ts

```diff
@@ -143,81 +143,81 @@ export type ValueSpecBoolean = {};
 export type ValueSpecAny =
   | Tag<"boolean", WithDescription<WithDefault<ValueSpecBoolean, boolean>>>
   | Tag<
-    "string",
-    WithDescription<WithDefault<WithNullable<ValueSpecString>, DefaultString>>
+      "string",
+      WithDescription<WithDefault<WithNullable<ValueSpecString>, DefaultString>>
   >
   | Tag<
-    "number",
-    WithDescription<WithDefault<WithNullable<ValueSpecNumber>, number>>
+      "number",
+      WithDescription<WithDefault<WithNullable<ValueSpecNumber>, number>>
   >
   | Tag<
-    "enum",
-    WithDescription<
-      WithDefault<
-        {
-          values: string[];
-          "value-names": {
-            [key: string]: string;
-          };
-        },
-        string
+      "enum",
+      WithDescription<
+          WithDefault<
+              {
+                  values: string[];
+                  "value-names"?: {
```

Comment:
:speech_balloon: Made the "value-names" conditional, because of how default works in serde and previous types of our config.yaml match this model better.

Reply:
it is only optional on the backend for backwards compatibility. It is preferred to be explicit